Build failure on armv7-l: "Stack layout invariants violated."

@staticfloat recently turned the buildbots for armv7-l back on, but the compilation currently fails with:

```
In file included from /buildworker/worker/package_linuxarmv7l/build/src/interpreter.c:9:0:
/buildworker/worker/package_linuxarmv7l/build/src/julia_internal.h:41:29: error: static assertion failed: "Stack layout invariants violated."
 #      define static_assert _Static_assert
                             ^
/buildworker/worker/package_linuxarmv7l/build/src/interpreter-stacktrace.c:263:1: note: in expansion of macro 'static_assert'
 static_assert(sizeof(interpreter_state) <= MAX_INTERP_STATE_SIZE,
 ^~~~~~~~~~~~~
make[1]: *** [interpreter.o] Error 1
```
https://build.julialang.org/#/builders/4/builds/797

This might a configuration issue of the buildbots, but maybe @yuyichao knows what is going on here.

--- a/src/interpreter-stacktrace.c
+++ b/src/interpreter-stacktrace.c
@@ -257,7 +257,7 @@ asm(
 
 #elif defined(_CPU_ARM_)
 
-#define MAX_INTERP_STATE_SIZE 32
+#define MAX_INTERP_STATE_SIZE 48
 
 // Check that the interpreter state can fit
 static_assert(sizeof(interpreter_state) <= MAX_INTERP_STATE_SIZE,

